base_rest/components/service.py

```diff
@@ -47,6 +47,15 @@ def skip_secure_params(func):
     func.skip_secure_params = True
 
 
+def skip_secure_response(func):
```

Comment:
@acsonefho You can remove the skip decorators.

Reply:
@lmignon The existing 'skip_secure_params' too?

Reply:
decorator**s** -> plural means also 'skip_secure_params' :smirk: